### ISSUE TITLE
After updating MacOSX 15.0 tableplus no longer works

### ISSUE THREAD

**Issue by Taffmailer** [TablePlus/TablePlus#3393]
Hello
I am using the latest Tableplus version for the Apple Silicon on a MacBook Pro M1 Max.
After the last MacOSX update to 15.0, I can no longer work with the software.
As soon as I switch to the “Queries” section after starting the program, it hangs. 

There are no further error messages.

regards
Franc

**Comment by ytorbyk** [comment]
@Taffmailer I had a similar issue about a month ago.
It was related to customer locations for "Connections", "Query/History" and "Other". They were configured to folders in Dropbox and it looks like Dropbox files became "on-demand".
Once I changed the configuration it resolved the issue.
Probably you have a similar cause of the issue.

**Comment by huyphams** [comment]
Correct, the on demand files/folders are the problems. TablePlus should do lazy load here (not blocking the UI).
